src/form.js

```diff
@@ -0,0 +1,159 @@
+import { addTasktoTasks} from "./tasks"
+import { taskLog } from "./taskLog"
+
+function formTitle(){
```

Comment:
Can be named "getFormTitle" so that it's purpose is more explicit. Can do that for the other functions too.